links/TalReverbLink/config.json

```diff
@@ -1,7 +1,8 @@
 {
     "label": "Tal Reverb 4",
     "class": "TalReverbLink",
-    "enabled": false,
+    "disabled": true,
```

Comment:
why do we need `disabled` versus just having both `disallowPreChain` and `disallowPostChain` both equal to `true`?